src/utils/envVars.ts

```diff
@@ -1,5 +1,5 @@
 export const IS_MAINNET                 = Boolean(process.env.REACT_APP_IS_MAINNET !== 'false');  // If REACT_APP_IS_MAINNET is unset, set it to true by default
-export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
+export const TESTNET_LAUNCHPAD_NAME     = process.env.REACT_APP_TESTNET_LAUNCHPAD_NAME || 'Pyrmont';
```

Comment:
Yeah, that was the only reason, to be able to navigate between them. This functionality didn't exist before (ie, no links to Pyrmont from mainnet)

More that I look at this, I do not see a reason to define `REACT_APP_ETH2_NETWORK_NAME` in the `.env` file at all. We already have a `REACT_APP_IS_MAINNET` boolean, which if true (or if `REACT_APP_TESTNET_LAUNCHPAD_NAME` isn't available) can force `ETH2_NETWORK_NAME` to "mainnet", like:
```
export const ETH2_NETWORK_NAME = (IS_MAINNET || !process.env.REACT_APP_TESTNET_LAUNCHPAD_NAME) ? 'mainnet' : process.env.REACT_APP_TESTNET_LAUNCHPAD_NAME;
```

We already build in logic looking for the word "mainnet":
```
process.env.REACT_APP_ETH2_NETWORK_NAME.toLowerCase() === "mainnet"
```
...then force this variable to "mainnet", the above would remove the need for `REACT_APP_ETH2_NETWORK_NAME` in `.env`

I have these changes working locally already. Thoughts @hwwhww @samajammin ?

Reply:
@wackerow I hear ya, I opened a similar PR. See: https://github.com/ethereum/eth2.0-deposit/pull/353

Carl's main reason is to be able to support multiple testnets.

Reply:
@samajammin Pushed this in 59fd7dd per our offline conversation, but just to document...

The method above should not prevent using a different testnet down the line. If Pyrmont is joined by a new testnet, the new testnet information can still be placed in the `.env` file, and it will replace any instance of "Pyrmont" on the page. This approach allows us to keep the new feature of being able to switch between networks, and removes redundant entries for the name of "mainnet", when "REACT_APP_IS_MAINNET" is enough.

Reply:
Note that we have to ask DevOps team (Jamie) to add the new env vars on Mainnet and Pyrmont servers **before** we merge the code into `master` branch.
/cc @CarlBeek